inventory/group_vars/all.yml

```diff
@@ -15,6 +20,9 @@ test:
   NAMESPACE: "{{ test_namespace }}"
   NAMESPACE_DISPLAY_NAME: "{{ test_namespace }}"
 
-role_binding_group:
+edit_role_binding_group:
   GROUP: "labs-ci-cd-contributors"
   ROLE: "{{ role | default('edit') }}"
+admin_role_binding_group:
```

Comment:
@springdo the group which is synced from IdM that all 'developers' belong to is called 'ocp-devs'